engine/runtime-instrument-common/src/main/java/org/enso/interpreter/service/ExecutionService.java

```diff
@@ -261,6 +262,24 @@ public void execute(
         onExecutedVisualizationCallback);
   }
 
+  /**
+   * Replace the missing arguments of the provided function call with {@link
+   * org.enso.interpreter.node.expression.builtin.Nothing} to make sure that the function call can
+   * be invoked.
+   *
+   * @param functionCall the function call to update
+   */
+  private void substituteMissingArguments(
+      FunctionCallInstrumentationNode.FunctionCall functionCall) {
+    var arguments = functionCall.getArguments();
```

Comment:
Didn't we wanted to change documentation of `getArguments()` method to mention:
- it can contain `null` values
- it's array is shared and shall not be modified in place?